pyramid_tm/__init__.py

```diff
@@ -50,6 +50,9 @@ def tm_tween(request):
             if not activate(request):
                 return handler(request)
 
+        # Set a flag in the environment to enable the `request.tm` property.
+        request.environ['tm.active'] = True
```

Comment:
there should be a guard against something else setting `tm.active` similar to the guard against `repoze.tm.active` as transaction cannot be nested